tests/acceptance/features/webUIResharing1/reshareUsers.feature

```diff
@@ -141,7 +141,7 @@ Feature: Resharing shared files with different permissions
     And the user opens folder "Shares" using the webUI
     Then the user should not be able to share resource "lorem.txt" using the webUI
 
-  @issue-5461
+  @skipOnOCIS
```

Comment:
This test is skipped because as mentioned in the issue https://github.com/owncloud/web/issues/6896, it also has a similar workflow of opening the `Shares` folder through UI and doing some action. Previously this test was expected to fail but they were linked to a closed issue.